src/cascadia/WinRTUtils/inc/WtExeUtils.h

```diff
@@ -45,9 +63,8 @@ _TIL_INLINEPREFIX bool IsDevBuild()
 // - <none>
 // Return Value:
 // - the full path to the exe, one of `wt.exe`, `wtd.exe`, or `WindowsTerminal.exe`.
-_TIL_INLINEPREFIX std::wstring GetWtExePath()
+_TIL_INLINEPREFIX const std::wstring& GetWtExePath()

```

Comment:
How are you getting away with returning a `wstring&` when it's from an object that you built on the stack in the body of this function?

Reply:
it's in the body of a static-call lambda right?

Reply:
Since it's a static object it also has a static address, which can then be returned safely.

Reply:
Oh, I just realized that I can annotate the other statics as const as well!

Reply:
Oh god my eyes blurred the lambda with the outer `const static`. OK OK. That makes more sense, yeesh. Sorry.